File: Dataset/Leetcode/train/1/235.py
 class Solution:
    def XXX(self, nums: List[int], target: int) -> List[int]:
        hash_map = {}
        for i in range(len(nums)):
            if nums[i] in hash_map:
                return [hash_map[nums[i]], i]
            hash_map.setdefault(target-nums[i], i)

File: Dataset/Leetcode/valid/75/235.cpp
class Solution {
public:
    void XXX(vector<int>& nums) {
        int a0=0,a1=0,a2=0;
        for(auto& num:nums){
            if(num==0)a0++;
            else if(num==1)a1++;
            else a2++;
        }
        for(int i=0;i<a0;i++)nums[i]=0;
        for(int i=a0;i<a0+a1;i++)nums[i]=1;
        for(int i=a0+a1;i<a0+a1+a2;i++)nums[i]=2;
 
    }
};

